core/src/test/java/com/linecorp/armeria/internal/server/annotation/ResponseConverterFunctionSelectorTest.java

```diff
@@ -0,0 +1,83 @@
+package com.linecorp.armeria.internal.server.annotation;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.ResponseConverterFunction;
+
+@SuppressWarnings("ConstantConditions")
+class ResponseConverterFunctionSelectorTest {
+
+    private static final ServiceRequestContext ctx = ServiceRequestContext.builder(
+            HttpRequest.of(HttpMethod.GET, "/")).build();
+
+    @Test
+    void prioritisesSpiDelegatingResponseConverterProvider() throws Exception {
+        final ResponseConverterFunction converterFunction =
+                ResponseConverterFunctionSelector.responseConverter(
+                        TestClassWithDelegatingResponseConverterProvider.class,
+                        Collections.singletonList(new MyResponseConverterFunction())
+                );
+
+        final HttpResponse response = converterFunction.convertResponse(ctx, null,
+                                                                        new TestClassWithDelegatingResponseConverterProvider(),
```

Comment:
Could you check the lint of test files? 😄

Reply:
Done 👍 Let me know if there are still issues, checkStyle seems to have passed